airbyte-ci/connectors/metadata_service/lib/pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "metadata-service"
-version = "0.24.1"
+version = "0.25.0"
```

Comment:
You will also want to update the orchestrator with these types.

e.g. this lock file: https://github.com/airbytehq/airbyte/blob/dae050c8fe3cf84909773eaa2b6d122227426583/airbyte-ci/connectors/metadata_service/orchestrator/poetry.lock

you can do this by running
```
cd airbyte-ci/connectors/metadata_service/orchestrator
poetry update metadata-service
```

Reply:
This change will cause ci to redeploy to dagster with the updated types

Reply:
Done, thanks for the insight!